build.gradle

```diff
@@ -32,6 +32,8 @@ dependencies {
     modImplementation "me.shedaniel.cloth:config-2:${project.clothconfig_version}"
     include "me.shedaniel.cloth:config-2:${project.clothconfig_version}"
     compileOnly "com.google.code.findbugs:jsr305:3.0.2"
+    modImplementation "io.github.prospector:modmenu:${project.mod_menu_version}"
```

Comment:
I don't see any reason to add ModMenu as dependency here.

Reply:
If not added, an error will be reported during compilation.
```
/home/plusls/github/minecraft-mods/Orderly/src/main/java/io/github/prospector/orderly/config/OrderlyModMenuCompat.java:3: error: package io.github.prospector.modmenu.api does not exist
import io.github.prospector.modmenu.api.ModMenuApi;
                                       ^
/home/plusls/github/minecraft-mods/Orderly/src/main/java/io/github/prospector/orderly/config/OrderlyModMenuCompat.java:10: error: cannot find symbol
public class OrderlyModMenuCompat implements ModMenuApi {
```

Reply:
ahh, right